Source/FolioReaderKit.swift

```diff
@@ -428,22 +442,6 @@ extension FolioReader {
     }
 }
 
-// MARK: - Application State
-
-@available(*, deprecated, message: "Use 'saveReaderState()' on a FolioReaderContainer object instead.")
-extension FolioReader {
-
-    /// Called when the application will resign active
-    open class func applicationWillResignActive() {
-        FolioReader.shared.saveReaderState()
```

Comment:
Maybe we should keep that for the legacy? I mean when people update the library they could/should get a simple warning. 
The function could be empty though, but at least it would still exist :)